Header header

# Lane Width [m]
float64 W
# lateral offset [m]
float64 Y_offset
# relative angle to lane [rad]
float64 dPhi
# Parabola coefficient [1]
float64 c0



